packages/training/deploy/docker/Dockerfile.base

```
@@ -0,0 +1,71 @@
+# Then use Dockerfile which extends this and builds in ~2 minutes.
+#
+# ============================================================================
+FROM vllm/vllm-openai:v0.14.0
+
+# Note: vllm/vllm-openai uses Python 3.12 and includes:
+# - PyTorch with CUDA
+# - vLLM with FlashInfer backend
+# - No flash-attn by default (uses FlashInfer instead - actually better for inference!)
+
+# Install system dependencies
+USER root
```

Hadolint (2.14.0)

[warning] Last USER should not be root

(DL3002)